release/models/acl/openconfig-acl.yang

```diff
@@ -830,6 +838,61 @@ module openconfig-acl {
     }
   }
 
+  grouping prefix-list-config {
+     leaf name {
+        type string;
+        description
+           "Name of the prefix list to
+            be used in the matches";
+      }
+      leaf description {
+        type string;
+        description
+          "Prefix list description";
+      }
```

Comment:
```suggestion
      }

```

Reply:
Here and throughout (I'll stop suggesting) can we have a blank line after each `leaf` to make it easier to read please.